src/vorta/keyring/secretstorage.py

```diff
@@ -15,55 +14,55 @@ class VortaSecretStorageKeyring(VortaKeyring):
 
     def __init__(self):
         """
-        Test whether DBus and Gnome-Keyring are available.
+        Test whether DBus and a SecretStorage provider are available.
         """
-        self.connection = secretstorage.dbus_init()
-        asyncio.set_event_loop(asyncio.new_event_loop())
-        secretstorage.get_default_collection(self.connection)
+        try:
+            self.connection = secretstorage.dbus_init()
+        except secretstorage.exceptions.SecretServiceNotAvailableException as e:
+            logger.debug("SecretStorage provider or DBus daemon is not available.")
+            raise e
+        else:
+            asyncio.set_event_loop(asyncio.new_event_loop())
+            self.collection = secretstorage.get_default_collection(self.connection)
```

Comment:
```suggestion
        asyncio.set_event_loop(asyncio.new_event_loop())
        self.collection = secretstorage.get_default_collection(self.connection)
```

The else is not needed since the raise will stop code execution.

Reply:
This was just my personal preference. But I've changed it nonetheless.

Reply:
This is cleaner code due to less branching.